### ISSUE TITLE
The generation was canceled 

### ISSUE THREAD

**Issue by Winddbourne** [Ivorforce/RecurrentComplex#274]
Every time I make and export a structure I get this message when I try to #gen it as a test. Do you have any idea what could be causing this?

![2017-09-18_17 24 18](https://user-images.githubusercontent.com/24589533/30565514-953cf774-9c96-11e7-89b5-b7bdd77bf6b9.png)

**Comment by Ivorforce** [comment]
This probably has to do with the Placer or an Ensure transformer. Further information should be in the logs (the message is a bit unspecific right now unfortunately).

**Comment by Winddbourne** [comment]
I managed to get my structures to generate but . . . it doesn't seem to generate static structures in other dimensions. I want to use structures to move between the overworld and dimensions created with JustEnoughDimensions and I'm wondering if I'm doing something wrong or if nothing is generated until a player actually enters a new dimension; including static (spawn) structures.

**Comment by Ivorforce** [comment]
Yes, structures are only being generated when a player enters the dimension (for performance reasons). You can look at the log for detailed information about when each structures is getting spawned.

If you don't enter the dimension, how are you event sure it generates at all though? I mean, you only require a structure to be present when you run across it normally.
